libraries/resource_firewall_rule.rb

```diff
@@ -18,7 +18,12 @@ class Resource::FirewallRule < Chef::Resource::LWRPBase
                            !!(p.to_s =~ /(udp|tcp|icmp|icmpv6|ipv6-icmp|none)/ || (p.to_s =~ /^\d+$/ && p.between?(0, 142)))
                          end })
     attribute(:direction, kind_of: Symbol, equal_to: [:in, :out, :pre, :post], default: :in)
-    attribute(:logging, kind_of: Symbol, equal_to: [:connections, :packets])
+
+    if Chef::Platform.windows?
```

Comment:
If we're not running on Windows, we should still allow someone to declare a Windows firewall rule. For example, if someone makes a`mysql` cookbook, I would want them to be able to ship Windows and Linux firewall rules, with `only_of { <platform_check> }`. This wouldn't compile if we have a conditional like what you've got here.